docs/how-to-use-gvfs.md

```diff
@@ -335,18 +335,17 @@ to recompile the native libraries like `libhdfs` and others, and completely repl
-| `cache_size`         | The cache capacity of the Gravitino Virtual File System.                                                                                                 | `20`          | No                                | 0.6.0         |.                                                                                                                |  (none)        | Yes                                 | 0.6.0         | .                               | (none)        | Yes      | 0.6.0         |
-| `cache_expired_time` | The value of time that the cache expires after accessing in the Gravitino Virtual File System. The value is in `seconds`.                                | `3600`        | No                                | 0.6.0         |.
-| `auth_type`          | The auth type to initialize the Gravitino client to use with the Gravitino Virtual File System. Currently supports `simple` and `oauth2` auth types.     | `simple`      | No                                | 0.6.0         |.
-| `oauth2_server_uri`  | The auth server URI for the Gravitino client when using `oauth2` auth type.                                                                              | (none)        | Yes if you use `oauth2` auth type | 0.7.0         |.
-| `oauth2_credential`  | The auth credential for the Gravitino client when using `oauth2` auth type.                                                                              | (none)        | Yes if you use `oauth2` auth type | 0.7.0         |.
-| `oauth2_path`        | The auth server path for the Gravitino client when using `oauth2` auth type. Please remove the first slash `/` from the path, for example `oauth/token`. | (none)        | Yes if you use `oauth2` auth type | 0.7.0         |.
-| `oauth2_scope`       | The auth scope for the Gravitino client when using `oauth2` auth type with the Gravitino Virtual File System.                                            | (none)        | Yes if you use `oauth2` auth type | 0.7.0         |.
-
+| Configuration item   | Description                                                                                                                                              | Default value | Required                          | Since version    |
+|----------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------|---------------|-----------------------------------|------------------|
+| `server_uri`         | The Gravitino server uri, e.g. `http://localhost:8090`.                                                                                                  | (none)        | Yes                               | 0.6.0-incubating |.                                                                                                                | (none)        | Yes                                 | 0.6.0         |
```

Comment:
Please remove this `0.6.0` here.

Reply:
![image](https://github.com/user-attachments/assets/65d23f95-add3-40a6-9dc3-b7877261609d)

Do you mean to write it like this on the line 'server uri'？

Reply:
I mean why there still version `0.6.0` in the document

Reply:
ok, i'll check my code

Reply:
There are some issues in the current doc, we just fixed the issue, @lsyulong can you please rebase your code, the comment here will be addressed automatically after you rebase the code against the main branch.

Reply:
The code has been revised.